### ISSUE TITLE
Training with keyframe conditioning

### ISSUE THREAD

**Issue by erenovic** [Lightricks/LTX-Video-Trainer#32]
Thank you for the great repository, the code is very tidy and easy to follow. However, I could not find details on keyframe conditioning (probably due to the timeline of LTX development). For a small project, I want to use the keyframe conditioning and finetune the LTX-Video v0.9.6 2B parameter model. Do you have any suggestions for how finetuning should work with keyframe conditioning? I followed the [LTX-Video pipeline](https://github.com/Lightricks/LTX-Video/blob/34625c3a1fb4e9a7d2f091938f6989816343e270/ltx_video/pipelines/pipeline_ltx_video.py#L1498C25-L1532C74) and concatenated the conditioning tokens while taking care of conditioning mask and pixel coordinates same as the inference code but results were not satisfactory. 

- Do you think if I should add any additional losses on the output conditioning tokens?
- I cannot see how concatenating latents for an arbitrary token can force consistency at same frame index?

Any pointers would be much appreciated :)

**Comment by zihaozou** [comment]
Hey @erenovic, you got any progress on this? I am also trying to implement the keyframe conditioning training

**Comment by erenovic** [comment]
Hello @zihaozou , unfortunately there is still no progress. I tried fine-tuning using LoRA but results were subpar.

**Comment by erenovic** [comment]
Hey @smirkun, unfortunately I switched to Wan2.1 as it had a clearer path for this. I also switched to [`diffusion-pipe`](https://github.com/tdrussell/diffusion-pipe) which already implemented a good deal of best practices if you would be interested.
